README.md

```diff
@@ -235,7 +235,7 @@ from apistar import routing
 def get_player_details(player_name):
     score = get_score(player_name)
     return {'name': player_name, 'score': score}
-    
+
```

Comment:
My autosave picked this up and removed the extra whitespace, not really in the context of this PR but should be ok